core/src/main/java/org/jdbi/v3/core/mapper/MapMapper.java

```diff
@@ -39,10 +49,13 @@ public MapMapper() {
 
     /**
      * Constructs a new MapMapper
-     * @param foldCase if true, column names are converted to lowercase in the mapped {@link Map}.
+     * @param toLowerCase if true, column names are converted to lowercase in the mapped {@link Map}.
+     *
+     * @deprecated use the {@link Config} class instead
      */
-    public MapMapper(boolean foldCase) {
-        this.foldCase = foldCase;
+    @Deprecated
```

Comment:
Let's not deprecate until the new API is out of beta

Reply:
done